README.md

```diff
@@ -247,9 +247,7 @@ See [Memoization](docs/Utilities.md#memoization) in the Utilities documentation
 
 ## Security
 
-This project is still in a very early and experimental phase. It has never been audited nor thoroughly reviewed for security vulnerabilities. Do not use in production.
```

Comment:
Not sure about removing this from the README. Maybe having it in both places?

Reply:
I was thinking about the same! Let's keep this line in `README.md` 👌